db.js

```diff
@@ -103,6 +115,8 @@ export class ActivityPubDB extends EventTarget {
     if (url && typeof url === 'object') {
       return url
     }
+    url = resolveP2PUrl(url)
```

Comment:
We shouldn't be trying to resolve to the gateway right away. Keep the current logic which resolve to the gateway only when the p2p logic fails.